README.md

```diff
@@ -61,8 +61,8 @@ library(usethis)
 # Create a new package -------------------------------------------------
 path <- file.path(tempdir(), "mypkg")
 create_package(path)
-#> ✔ Creating '/tmp/Rtmpztuf0Z/mypkg/'
-#> ✔ Setting active project to '/private/tmp/Rtmpztuf0Z/mypkg'
+#> ✔ Creating '/var/folders/5r/wrwbj0sn3m71clq0l1rm4pmm0000gp/T/Rtmpt7SYWI/mypkg/'
+#> ✔ Setting active project to '/private/var/folders/5r/wrwbj0sn3m71clq0l1rm4pmm0000gp/T/Rtmpt7SYWI/mypkg'
```

Comment:
This is low-stakes and the ease depends on your git client/workflow. But I often discard these nuisance changes when rebuilding README.

Reply:
Yeah, I often do as well...